modules/Bio/EnsEMBL/Analysis/Hive/Config/GenblastHomology.pm

```diff
@@ -0,0 +1,532 @@
+    'genblast_db_port'             => $self->o('databases_port'),
+
+
+    'killlist_db_server'           => $self->o('databases_server'),
+    'killlist_db_port'             => $self->o('databases_port'),
+
+    # This is used for the ensembl_production and the ncbi_taxonomy databases
+    'ensembl_release'              => $ENV{ENSEMBL_RELEASE}, # this is the current release version on staging to be able to get the correct database
+
+    databases_to_delete => ['genblast_db'],
+
+########################
+# BLAST db paths
+########################
+    'base_blast_db_path'        => $ENV{BLASTDB_DIR},
+
+######################################################
+#
+# Mostly constant settings
+#
+######################################################
+
+    genome_dumps                  => catdir($self->o('output_path'), 'genome_dumps'),
+    # This one is used by most analyses that run against a genome flatfile like exonerate, genblast etc. Has slice name style headers. Is softmasked
+    softmasked_genome_file        => catfile($self->o('genome_dumps'), $self->o('species_name').'_softmasked_toplevel.fa'),
+    # This one is used in replacement of the dna table in the core db, so where analyses override slice->seq. Has simple headers with just the seq_region name. Also used by bwa in the RNA-seq analyses. Not masked
+
+    'min_toplevel_slice_length'   => 250,
+
+    'homology_models_path'        => catdir($self->o('output_path'),'homology_models'),
+
+    ensembl_analysis_script           => catdir($self->o('enscode_root_dir'), 'ensembl-analysis', 'scripts'),
+    remove_small_orf_script             => catfile($self->o('ensembl_analysis_script'), 'genebuild', 'remove_small_orf.pl'),
+  
+    # cutoffs for removing small_orf genes
+    'small_orf_cutoff' => '100',
+    'intron_cutoff' => '75',
+
+########################
+# Extra db settings
+########################
+
+    'num_tokens' => 10,
+    mysql_dump_options => '--max_allowed_packet=1000MB',
+
+########################
+# Executable paths
+########################
+
+    'blast_type' => 'ncbi', # It can be 'ncbi', 'wu', or 'legacy_ncbi'
+    'exonerate_path'         => catfile($self->o('software_base_path'), 'opt', 'exonerate09', 'bin', 'exonerate'),
+    'uniprot_genblast_batch_size' => 15,
+    'uniprot_table_name'          => 'uniprot_sequences',
+    genewise_path => catfile($self->o('binary_base'), 'genewise'),
+    'genblast_path'     => catfile($self->o('binary_base'), 'genblast'),
+    'genblast_eval'     => $self->o('blast_type') eq 'wu' ? '1e-20' : '1e-1',
+    'genblast_cov'      => '0.5',
+    'genblast_pid'      => '30',
+    'genblast_max_rank' => '5',
+    'genblast_flag_small_introns' => 1,
+    'genblast_flag_subpar_models' => 0,
+
+
+##################################
+# Memory settings for the analyses
+##################################
+    'default_mem'          => '900',
+    'genblast_mem'         => '1900',
+    'genblast_retry_mem'   => '4900',
```

Comment:
This can be removed, memory is set in the resource classes

Reply:
memory options removed